### PR TITLE
Convert examples/MessageGroups to Swift 3.0

### PR DESCRIPTION
**Pull request by nixzhu**

### CODE REVIEW

File: examples/MessageGroups/Podfile [hunk 1]
```diff
@@ -19,3 +21,11 @@ target 'MessageGroups' do
...
 
 end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['SWIFT_VERSION'] = '3.0'
+    end
+  end
+end
```
Comment (atainter) [inline examples/MessageGroups/Podfile]: What is this used for?

File: examples/MessageGroups/Podfile [hunk 2]
```diff
@@ -5,8 +5,10 @@ target 'MessageGroups' do
   # Comment this line if you're not using Swift and don't want to use dynamic frameworks
   use_frameworks!
 
+  swift_version = '3.0'
+
```
Comment (atainter) [inline examples/MessageGroups/Podfile]: Is this used?

**Comment by nixzhu** [comment]
CocoaPods default set pod's swift version to 2.3, so before `pod trunk
push` your pod, you need a `.swift_version` file to specify it. These code
is a workaround for local pod without the file.

Aaron Tainter <notifications@github.com>于2016年9月30日 周五04:39写道：

**Comment by atainter** [comment]
LG

**Pull request merged**